annotator/src/main/java/org/cbioportal/annotator/internal/GenomeNexusImpl.java

```diff
@@ -684,7 +684,14 @@ private String getVariantClassificationFromMap(String variant) {
             variantMap.put("feature_elongation",            "Targeted_Region"); 
             variantMap.put("feature_truncation",            "Targeted_Region"); 
         }
-        
+        if (variant.toLowerCase().equals("protein_altering_variant") || variant.toLowerCase().equals("coding_sequence_variant")) {
+            if (resolveVariantType().equals("DEL")) {
+                return "In_Frame_Del";
+            }
+            else if (resolveVariantType().equals("INS")) {
```

Comment:
We are calling the `resolveVariantType` method twice here. If it is a costly operation, it might be better to call it once and assign it to a variable.